plugin_config.txt

```diff
@@ -1692,6 +1692,52 @@ my $VEP_PLUGIN_CONFIG = {
       ]
     },
 
+    # mutfunc
+    {
+      "key" => "mutfunc",
+      "label" => "mutfunc",
+      "available" => 0,
+      "enabled" => 0,
+      "section" => "Protein annotation",
+      "helptip" => "mutfunc predicts destabilization effect of protein structure, interaction, regulatory region etc. caused by a variant",
+      "plugin_url" => "https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/108/mutfunc.pm",
+      "requires_data" => 1,
+      "params"  => [
+        "#db=/path/to/mutfunc_human_data.db",
+        "@*"
+      ],
+      "species" => [
+        "homo_sapiens",
+        "saccharomyces_cerevisiae"
+      ],
+      "form" => [
+        {
+          "name" => "mutfunc_fields",
+          "label" => "Fields to include",
+          "helptip" => "Select which analysis you want to see in the output",
+          "value" => "",
+          "type" => "checklist",
+          "multiple" => 1,
+          "style" => "height:150px",
+          "values" => [
+            { "value" => "motif=1", "caption" => "motif", "helptip" => "Effect on linear motifs", 'checked' => 1 },
+            { "value" => "int=1", "caption" => "int", "helptip" => "Effect on protein interactions", 'checked' => 1 },
+            { "value" => "mod=1", "caption" => "mod", "helptip" => "Effect on protein structure", 'checked' => 1 },
+            { "value" => "exp=1", "caption" => "exp", "helptip" => "Effect on protein structure (experimental)", 'checked' => 1 },
+            { "value" => "tfbs=1", "caption" => "tfbs", "helptip" => "Effect on transcription factor binding site", 'checked' => 1 },
+          ],
```

Comment:
It would be good to make the main name a bit more descriptive so that the user isn't reliant on tool tips. Something like:

motif -> Linear Motifs
int -> Protein Interactions
mod -> Protein structure
exp -> Protein structure (exp.)
tfbs -> Transcription factor binding

Reply:
Thanks @jamie-m-a , updated the name and also removed tfbs which currently not supported.